src/assets/JSAsset.js

```diff
@@ -116,7 +116,8 @@ class JSAsset extends Asset {
     }
 
     return {
-      js: code
+      js: code,
+      map: this.sourcemap
```

Comment:
We'll also need to generate 1:1 source maps for files that we don't process with babel.

Reply:
Will work on that once the JS maps are more on point

Reply:
done